packages/stores/src/queries/superfluid-pools/types.ts

```diff
@@ -0,0 +1,89 @@
+import { CoinPretty } from "@keplr-wallet/unit";
+import { Duration } from "dayjs/plugin/duration";
+
+export type SuperfluidParams = {
+  params: {
+    // Dec
```

Comment:
I'm not sure. I literally copied the whole superfluid store directory. I'll look for areas it could improve. 

I'm assuming this that is correct, this is the shape of the API data, the comment is the type it should be converted to for the frontend.